src/components/TranslateArea.js

```diff
@@ -6,10 +6,10 @@ function TranslateArea() {
     const [message, setMessage] = useState('');
     const translation = useContext(TranslationContext);
 
-    const alBhed = ['E', 'P', 'S', 'T', 'I', 'W', 'K', 'N', 'U', 'V', 'G', 'C', 'L', 'R', 'Y', 'B', 'X', 'H', 'M', 'D', 'O', 'F', 'Z', 'Q', 'A', 'J', 'e', 'p', 's', 't', 'i', 'w', 'k', 'n', 'u', 'v', 'g', 'c', 'l', 'r', 'y', 'b', 'x', 'h', 'm', 'd', 'o', 'f', 'z', 'q', 'a', 'j'];
-    const bhedAl = ['Y', 'P', 'L', 'T', 'A', 'V', 'K', 'R', 'E', 'Z', 'G', 'M', 'S', 'H', 'U', 'B', 'X', 'N', 'C', 'D', 'I', 'J', 'F', 'Q', 'O', 'W', 'y', 'p', 'l', 't', 'a', 'v', 'k', 'r', 'e', 'z', 'g', 'm', 's', 'h', 'u', 'b', 'x', 'n', 'c', 'd', 'i', 'j', 'f', 'q', 'o', 'w']
```

Comment:
The compiler was warning that defining these arrays outside of the translateAlBhed callback function was causing unnecessary warnings, so I moved them inside of the useCallback call and everything seemed okay (definitely double-check the behavior of the app).

Reply:
That interesting.  The behavior is the same though.  Great catch, I completely missed the warning.